magpie/utils.py

```diff
@@ -174,8 +172,7 @@ def convert_response(response):
         for cookie in response.cookies:
             pyramid_response.set_cookie(name=cookie.name, value=cookie.value, overwrite=True)
     if isinstance(response, HTTPException):
-        # noinspection PyProtectedMember
-        for header_name, header_value in response.headers._items:
+        for header_name, header_value in response.headers._items:  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _items of a client class](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)